js/exercise-00.js

```diff
@@ -0,0 +1,27 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+*/
+
+
+
+const String = "Yasir";
```

Comment:
Variables are defined using camelCase convention. Don't use uppercase to name them.

```suggestion
const string = "Yasir";
```

FIX: rename properly variables